### DISCUSSION TITLE
Not showing all photos when the photos count is big.

### DISCUSSION THREAD

**Discussion by mexusbg** [photoprism/photoprism#2012] [Q&A] [answered]
Not showing all the latest photos / showing in random photos first.

Last version.
About 10000 images in my folder.

Rescan doesn't fix the issue

**Comment by graciousgrey** [comment, marked as answer]
Please follow our issue template and only report clearly identified bugs. Make sure to read the troubleshooting guide as well: https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures
